scripts/delete_sqs_queues.py

```diff
@@ -27,26 +50,28 @@ def get_queue_attributes(queue_name):
         ]
     )
     queue_attributes = response['Attributes']
+    queue_attributes.update({'QueueUrl': queue_name})
     return queue_attributes
 
 
-def delete_queue(queue_name):
+def delete_queue(queue_url):
+    print("DELETEING {}".format(queue_url))
     response = client.delete_queue(
-        QueueUrl=queue_name
+        QueueUrl=queue_url
     )
     if response['ResponseMetadata']['HTTPStatusCode'] == 200:
```

Comment:
I should've added this in as a comment, but apparently according to `boto` docs for `delete_queue` it says that "If the specified queue doesn't exist, Amazon SQS returns a successful response.". Just worth adding in as a comment for future users of the script.